frontend/src/views/invite/InviteView.vue

```diff
@@ -41,29 +33,31 @@ const detail: Ref<string> = ref('Please check your link again');
 // Actions
 const toast = useToast();
 
-const errorMessage: ErrorType = {
-  401: {
-    title: 'Unauthorized',
-    detail: 'Invalid authorization credentials'
-  },
-  403: {
-    // wrong email
-    title: 'Forbidden',
-    detail: 'The person who sent you the link may have restricted it to another email address'
-  },
-  404: {
-    // invitation not found/expired
-    title: 'Not found',
-    detail: 'Invitation not found or has expired'
-  },
-  409: {
-    // bucket/object not found
-    // TODO: display specifically object or bucket
-    title: 'Not found',
-    detail: 'Invalid object or bucket'
+const setErrorMessage = (errorCode: number): void => {
+  switch (errorCode) {
+    case 401:
+      title.value = 'Unauthorized';
+      detail.value = 'Invalid authorization credentials';
+      break;
+    case 403:
+      title.value = 'Forbidden';
+      detail.value = 'The person who sent you the link may have restricted it to another email address';
+      break;
+    case 404:
+      title.value = 'Not found';
+      detail.value = 'Invitation not found or has expired';
+      break;
+    case 409:
+      title.value = 'Not found';
+      detail.value = 'Invalid object or bucket';
+      break;
+    case 410:
+      title.value = 'Gone';
+      detail.value = 'Invitation has expired';
+      break;
+    default:
```

Comment:
We need a default fallback value title and detail error message for when the error code is not one of the above (for some reason or another). I suggest the following:
```ts
default:
  title.value = 'Error';
  detail.value = 'Something has gone horribly wrong :(';
  break;
```